pylintrc

```diff
@@ -171,7 +171,7 @@ single-line-if-stmt=no
 no-space-check=trailing-comma,dict-separator
 
 # Maximum number of lines in a module
-max-module-lines=2000
+max-module-lines=1500
```

Comment:
This does not seem to work.

Reply:
I'm not a huge fan of hard limits. Sometimes it just makes more sense to keep things together even if that would be a few lines over the limit.

Reply:
Can you remove the change here? If at all, it's probably better to do that separately.